torch/csrc/jit/tensorexpr/llvm_codegen.cpp

```diff
@@ -1374,16 +1407,15 @@ void LLVMCodeGenImpl::visit(const Intrinsics* v) {
         SIMD_BINARY_MATH_CASE(kFmod, "fmodf", FloatTy_)
 #undef SIMD_BINARY_MATH_CASE
 
-#define BINARY_MATH_CASE(enum, name, type)                             \
-  case enum: {                                                         \
-    auto callee = module_->getOrInsertFunction(                        \
-        name, llvm::FunctionType::get(type, {type, type}, false), {}); \
-    call_ty = callee.getFunctionType();                                \
-    call_fn = callee.getCallee();                                      \
-    applyMathFunctionAttributes(llvm::cast<llvm::Function>(call_fn));  \
-  } break;
-        BINARY_MATH_CASE(kRemainder, "remainderf", FloatTy_)
-#undef BINARY_MATH_CASE
+      case kRemainder: {
```

Comment:
Oh gee, that used to be pretty convoluted.

Reply:
Hah, yeah I think it was done for consistency with some of the above cases, but... I actually removed the macro because I had a really dumb error (using `auto` instead of `FunctionCallee` was messing up the types) but I couldn't really grok the error because of how the macro was obscuring the compiler error.